galaxy_importer/collection.py

```diff
@@ -173,12 +176,56 @@ def _load_collection_manifest(self):
         if not os.path.exists(manifest_file):
             raise exc.ManifestNotFound('No manifest found in collection')
 
+        default_logger.debug('manifest_file: %s', manifest_file)
+
         with open(manifest_file, 'r') as f:
             try:
                 data = schema.CollectionArtifactManifest.parse(f.read())
             except ValueError as e:
                 raise exc.ManifestValidationError(str(e))
             self.metadata = data.collection_info
+            default_logger.debug('data.file_manifest_file: %s', data.file_manifest_file)
+            self.file_manifest_file = data.file_manifest_file
+
+    def _load_collection_file_manifest(self):
+        default_logger.debug('self.file_manifest_file: %s', self.file_manifest_file)
+
+        files_manifest_file = os.path.join(self.path, self.file_manifest_file.name)
+        if not os.path.exists(files_manifest_file):
+            raise exc.ManifestNotFound(f'No file manifest file ({self.file_manifest.name}) found in collection')
```

Comment:
can this be a generic `ImporterError` or something different? if a collection has a `MANIFEST.json` I dont think we should throw a `ManifestNotFound` error

Reply:
Hmm, yeah.  The latest version moves the check for the actual file into collections.check_artifact_file()
but the error raised from there will be a:

`CollectionArtifactFileNotFound("The file (FILES.json) was listed in the file manifest but it was not found")`

Which is kind of worse for the particular case of FILES.json.  I'll see if I can simplify

Reply:
I removed the somewhat extraneous details about being listed in the file manifest from the error.